tests/models/test_models.py

```diff
@@ -6,39 +6,41 @@
 
 MODELS = [
     "facebook/opt-125m",
```

Comment:
you'll restore this change right?

Reply:
ah yeah it is for quick testing. reverting